test/functional/ContextAwareInterfaceTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Dhii\Output\FuncTest\FuncTest;
+
+use Dhii\Output\ContextAwareInterface;
+use Xpmock\TestCase;
+
+/**
+ * Tests {@see \Dhii\Output\FuncTest\ContextAwareInterface}.
+ *
+ * @since [*next-version*]
+ */
+class ContextAwareInterfaceTest extends TestCase
+{
+    /**
+     * The class name of the test subject.
+     *
+     * @since [*next-version*]
+     */
+    const TEST_SUBJECT_CLASSNAME = 'Dhii\Output\FuncTest\ContextAwareInterface';
+
+    /**
+     * Creates a new instance of the test subject.
+     *
+     * @since [*next-version*]
+     *
+     * @return ContextAwareInterface
+     */
+    public function createInstance()
+    {
+        $mock = $this->mock(static::TEST_SUBJECT_CLASSNAME)
+                     ->getContext()
+                     ->new();
+
+        return $mock;
+    }
+
+    /**
+     * Tests whether a valid instance of the test subject can be created.
+     *
+     * @since [*next-version*]
+     */
+    public function testCanBeCreated()
+    {
+        $subject = $this->createInstance();
+
+        $this->assertInstanceOf(
+            static::TEST_SUBJECT_CLASSNAME,
+            $subject,
+            'Subject is not a valid instance'
```

Comment:
"A valid instance of the subject could not be created" please. I explained this before. Subject is a valid instance, in that it is an instance. But it is not an instance of the subject.